README.md

```diff
@@ -22,7 +22,7 @@ Special thanks to everyone who contributed!
 |---|---|---|---|---|
  | [<img src="https://avatars3.githubusercontent.com/u/446112?v=4" width="100px" /><br /><sub>pedrovitti</sub>](https://github.com/pedrovitti) | [<img src="https://avatars0.githubusercontent.com/u/1775157?v=4" width="100px" /><br /><sub>LeonardoCardoso</sub>](https://github.com/LeonardoCardoso) | [<img src="https://avatars1.githubusercontent.com/u/88917?v=4" width="100px" /><br /><sub>nicbou</sub>](https://github.com/nicbou) | [<img src="https://avatars1.githubusercontent.com/u/29275608?v=4" width="100px" /><br /><sub>gpodolan</sub>](https://github.com/gpodolan) | [<img src="https://avatars1.githubusercontent.com/u/112706?v=4" width="100px" /><br /><sub>luizkowalski</sub>](https://github.com/luizkowalski)
  | [<img src="https://avatars0.githubusercontent.com/u/237513?v=4" width="100px" /><br /><sub>drubin</sub>](https://github.com/drubin) | [<img src="https://avatars2.githubusercontent.com/u/450069?v=4" width="100px" /><br /><sub>mathiasgrimm</sub>](https://github.com/mathiasgrimm) | [<img src="https://avatars3.githubusercontent.com/u/2914703?v=4" width="100px" /><br /><sub>DhiaTN</sub>](https://github.com/DhiaTN) | [<img src="https://avatars1.githubusercontent.com/u/502575?v=4" width="100px" /><br /><sub>fernandosouza</sub>](https://github.com/fernandosouza) | [<img src="https://avatars2.githubusercontent.com/u/1649432?v=4" width="100px" /><br /><sub>frontendherodk</sub>](https://github.com/frontendherodk)
- | [<img src="https://avatars3.githubusercontent.com/u/36678?v=4" width="100px" /><br /><sub>duduBaiao</sub>](https://github.com/duduBaiao) | [<img src="https://avatars0.githubusercontent.com/u/3342195?v=4" width="100px" /><br /><sub>ehkenne</sub>](https://github.com/ehkenne) | [<img src="https://avatars0.githubusercontent.com/u/26120299?v=4" width="100px" /><br /><sub>danieladelvos</sub>](https://github.com/danieladelvos)
+ | [<img src="https://avatars3.githubusercontent.com/u/36678?v=4" width="100px" /><br /><sub>duduBaiao</sub>](https://github.com/duduBaiao) | [<img src="https://avatars0.githubusercontent.com/u/3342195?v=4" width="100px" /><br /><sub>ehkenne</sub>](https://github.com/ehkenne) | [<img src="https://avatars0.githubusercontent.com/u/26120299?v=4" width="100px" /><br /><sub>danieladelvos</sub>](https://github.com/danieladelvos) | [<img src="https://avatars3.githubusercontent.com/u/404460?v=4" width="100px" /><br /><sub>iamkhush</sub>](https://github.com/iamkhush)
```

Comment:
This is automatically generated, you don't need to include yourself in the list of contributors

Reply:
Alright.